test/integration/agent/openflow_test.go

```diff
@@ -100,6 +102,12 @@ func TestConnectivityFlows(t *testing.T) {
 	// Initialize ovs metrics (Prometheus) to test them
 	metrics.InitializeOVSMetrics()
 
+	// Hack the OS type if we run the test not on Windows Node.
+	// Because we test some Windows only functions.
+	if !env.IsWindowsPlatform() {
```

Comment:
This is tricky and doesn't solve the problem as it tests windows only now.

Could `generatePipeline` be covered by unit test instead? I think it would be much easier to cover various scenarios.